source/_integrations/mullvad.markdown

```diff
@@ -16,6 +16,12 @@ ha_platforms:
 
 The Mullvad VPN integration pulls [Mullvad](https://mullvad.net/) VPN status information into Home Assistant.
 
+<div class='note'>
+
+This integration reports the Mullvad status of Home Assistant itself, not the status of other clients that may be connected to your Mullvad account.
+
+</div>
```

Comment:
I suggest we not make this a big fat not, but instead, incorporate this into the introduction text normally.